captura/views.py

```diff
@@ -858,6 +857,29 @@ def update(self, request, pk):
         else:
             return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
 
+    def delete(self, request, pk):
+        """ Soft deletes a specific instance of a Study.
+
+            Raises
+            ------
+            HTTP STATUS 404
+            If the study does not exist or it does not belong to the capturista.
+
+            Returns
+            -------
+            On Success
+                Response 200
+            On Error
+                Response 404
+        """
+        queryset = Estudio.objects.filter(capturista=request.user.capturista)
+        estudio = get_object_or_404(queryset, pk=pk)
+
+        if estudio.status == Estudio.BORRADOR:
```

Comment:
All related family members should also be soft-deleted as with the regular delete view of a study.